Makefile

```diff
@@ -60,24 +79,35 @@ else
 	BUILDTAGS ?= seccomp apparmor
 endif
 
+IMAGE_BASE_DISTRO := $(shell lsb_release -is | tr '[:upper:]' '[:lower:]')
+
 # Identify kernel-headers path if not previously defined. Notice that this logic is already
 # present in Sysbox's Makefile; we are duplicating it here to keep sysbox-runc as independent
 # as possible. If KERNEL_HEADERS is not already defined, we will assume that the same applies
 # to all related variables declared below.
-ifeq ($(KERNEL_HEADERS),)
-	IMAGE_BASE_DISTRO := $(shell lsb_release -is | tr '[:upper:]' '[:lower:]')
-	ifeq ($(IMAGE_BASE_DISTRO),$(filter $(IMAGE_BASE_DISTRO),centos fedora redhat))
-		KERNEL_HEADERS := kernels/$(KERNEL_REL)
-	else
-		KERNEL_HEADERS := linux-headers-$(KERNEL_REL)
-		KERNEL_HEADERS_BASE := $(shell find /usr/src/$(KERNEL_HEADERS) -maxdepth 1 -type l -exec readlink {} \; | cut -d"/" -f2 | egrep -v "^\.\." | head -1)
-	endif
-	ifeq ($(KERNEL_HEADERS_BASE), )
-		KERNEL_HEADERS_MOUNTS := -v /usr/src/$(KERNEL_HEADERS):/usr/src/$(KERNEL_HEADERS):ro
-	else
-		KERNEL_HEADERS_MOUNTS := -v /usr/src/$(KERNEL_HEADERS):/usr/src/$(KERNEL_HEADERS):ro \
-					 -v /usr/src/$(KERNEL_HEADERS_BASE):/usr/src/$(KERNEL_HEADERS_BASE):ro
+ifeq ($(IMAGE_BASE_DISTRO),$(filter $(IMAGE_BASE_DISTRO),centos fedora redhat))
```

Comment:
That addition was in the top level Makefile; but I think it makes sense to add it to sysbox-runc Makefile too (otherwise sysbox-runc unit tests on an AlmaLinux host won't work).

Reply:
Yep, that's what i meant.

Reply:
I added almalinux to the list.